frontend/src/app/AboutDialog.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {
+  AboutModal,
+  Alert,
+  Bullseye,
+  Spinner,
+  TextContent,
+  TextList,
+  TextListItem,
+} from '@patternfly/react-core';
+import { ODH_LOGO, ODH_PRODUCT_NAME } from '~/utilities/const';
+import { useUser, useClusterInfo } from '~/redux/selectors';
+import { useAppContext } from '~/app/AppContext';
+import useFetchDsciStatus from '~/concepts/areas/useFetchDsciStatus';
+import { useWatchOperatorSubscriptionStatus } from '~/utilities/useWatchOperatorSubscriptionStatus';
+
+import './AboutDialog.scss';
+
+const RhoaiAboutText = `Red Hat® OpenShift® AI (formerly Red Hat OpenShift Data Science) is a flexible, scalable MLOps platform for data scientists and developers of artificial intelligence and machine learning (AI/ML) applications. Built using open source technologies, OpenShift AI supports the full lifecycle of AI/ML experiments and models, on premise and in the public cloud.`;
+const RhoaiDefaultReleaseName = `OpenShift AI`;
+
+const OdhAboutText = `Open Data Hub is an open source AI platform designed for the hybrid cloud. The community seeks to bridge the gap between application developers, data stewards, and data scientists by blending the leading open source AI tools with a unifying and intuitive user experience. Open Data Hub supports the full lifecycle of AI/ML experiments and models.`;
+const OdhDefaultReleaseName = `Open Data Hub`;
```

Comment:
@andrewballantyne are you ok with these hard coded values? Is it not possible to source these dynamically?

Reply:
The release names here are fallback values when `dsciStatus?.release?.name` is undefined.